src/util.c

```diff
@@ -633,3 +633,25 @@ All facets connected.  No further nearby check necessary.\n");
     stl_verify_neighbors(stl);
   }
 }
+
+int stl_check_results(stl_file *stl, int reverse_all_flag){
+
+  if(
+    stl->stats.facets_w_1_bad_edge != 0 ||
+    stl->stats.facets_w_2_bad_edge != 0 ||
+    stl->stats.facets_w_3_bad_edge != 0 ||
+    stl->stats.degenerate_facets   != 0 ||
+    stl->stats.edges_fixed         != 0 ||
+    stl->stats.facets_removed      != 0 ||
+    stl->stats.facets_added        != 0 ||
+    stl->stats.backwards_edges     != 0 ||
+    stl->stats.normals_fixed       != 0 ||
+    (
+      !reverse_all_flag            &&
+      stl->stats.facets_reversed   != 0
+    )
+  )
+    return 1;
+  else
+    return 0;
```

Comment:
Two questions:

 - Can the `if conditional: return 1, else return 0` construct be replaced with `return conditional` in C?
 - What does the function return? It returns 1 (aka true) when the STL was invalid? In that case should the name reflect that? `stl_check_results` doe snot really tell which way it works. I would expect it to return 1 when the STl was valid. We could either flip the values or rnema it, e.g. to `stl_was_invalid`.

Reply:
First question: yes, good point. I actually don't know why I thought it would me more readable inside the if statement.
Second question: I will rename the function to `stl_is_invalid` - because it checks current STL stats. Would that make more sense?